components/ProductGrid/components/Item/index.jsx

```diff
@@ -16,110 +17,117 @@ import FavoritesButton from '@shopgate/pwa-ui-shared/FavoritesButton';
 import styles from './style';
 import connect from './connector';
 
+const props = {};
```

Comment:
Why is this inside this ticket? We have no good way to test this out yet.

Reply:
Done